src/components/InputTodo.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+
+// eslint-disable-next-line react/prop-types
```

Comment:
- Kindly remove eslint disable lines from your project as it is not a good coding practice.